app/models/merchant.rb

```diff
@@ -45,11 +45,36 @@ def items_ready_to_ship
     .order('created_at')
   end
 
+  # def self.best_day(merchant)
+  #   find_by_sql(
+  #     "select invoices.created_at, sum(invoice_items.quantity * invoice_items.unit_price) as revenue from invoice_items
+  #       inner join invoices on invoices.id = invoice_items.invoice_id
+  #       inner join items on items.id = invoice_items.item_id
+  #       inner join merchants on merchants.id = items.merchant_id
+  #       inner join transactions on invoices.id = transactions.invoice_id
+  #     where merchants.id = #{merchant.id}
+  #     and transactions.result = 1
+  #     group by invoices.created_at
+  #     order by revenue desc
+  #     limit 1;"
+  #   )
+  # end
```

Comment:
@deadbert I wanted to check in before removing these commented out lines. Let me know if I can remove them and I will commit it in this PR.